docs/installation/issues.md

```diff
@@ -131,6 +131,15 @@ As is suggested by starfry's comment on stackoverflow [http://stackoverflow.com/
 As per the documentation for [rspec-rails](https://www.relishapp.com/rspec/rspec-rails/docs),
   > It needs to be in the :development group to expose generators and rake tasks without having to type RAILS_ENV=test.
 
+### Ruby version missmatch
+
+If you're running Mac OsX and High Sierra, you may get a Ruby version missmatch error when trying to run 
```

Comment:
May want to include a snippet of the exact error you received

Reply:
good point!